Web-Project.md

```diff
@@ -122,27 +122,88 @@ These settings assume that Pyramid is installed into a virtual environment at ".
 
 If you have settings for another framework that you would like to share, or if you'd like to request settings for another framework, visit our [issue tracker](https://github.com/Microsoft/PTVS/issues) and start a post.
 
-Publishing to Microsoft Azure
------------------------------
+Publishing to Microsoft Azure App Service
+-----------------------------------------
 
```

Comment:
Templates section above also needs an update.

"When deploying to Microsoft Azure, using a virtual environment based on 32-bit Python 2.7 or 3.4 is required to ensure the deployed app has the same libraries as when run locally"

Also it refers to Azure Web Sites instead of App Service.